specs/DotNetProjectFile.Analyzers.Specs/Rules/Generic/Only_use_UTF-8_without_BOM.cs

```diff
@@ -0,0 +1,24 @@
+using DotNetProjectFile.Analyzers.Files;
+
+namespace Rules.Generic.Only_use_UTF8_without_BOM;
+
+public class Reports
+{
+    [Test]
+    public void files_with_UTF8_BOM()
+       => new OnlyUseUTF8WithoutBom()
+       .ForProject("MultipleEncodings.cs")
+       .HasIssues(
+           new Issue("Proj3000", "This file's encoding using UTF-8 with BOM.").WithPath("utf8-bom.css"),
+           new Issue("Proj3000", "This file's encoding using UTF-8 with BOM.").WithPath("file.utf8-bom"));
+}
+
+public class Guards
+{
+    [TestCase("CompliantCSharp.cs")]
```

Comment:
Can you add an explicit utf8 without bom encoded file to this project

Reply:
The project files themselves are using UTF-8 without BOM.